homeassistant/components/filter/sensor.py

```diff
@@ -434,14 +437,21 @@ def __init__(
         else:
             self.states = deque(maxlen=0)
             self.window_unit = WINDOW_SIZE_UNIT_TIME
-        self.precision = precision
+        self._precision = precision
         self._name = name
         self._entity = entity
         self._skip_processing = False
         self._window_size = window_size
         self._store_raw = False
         self._only_numbers = True
 
+    @property
+    def precision(self):
```

Comment:
I suggest that you rename this property to avoid clash with #86074
```suggestion
    def filter_precision(self):
```